prog/postgres/postgres_server_nexus.rb

```diff
@@ -204,7 +204,7 @@ def before_run
   scrape_interval: 10s
   external_labels:
     ubicloud_resource_id: #{postgres_server.resource.ubid}
-    ubicloud_resource_role: #{(postgres_server.id == postgres_server.resource.representative_server.id) ? "primary" : "standby"}
+    ubicloud_resource_role: #{(!postgres_server.read_replica? && (postgres_server.id == postgres_server.resource.representative_server.id)) ? "primary" : "standby"}
```

Comment:
In this context, shouldn't we send the role as `primary` if the server is representative_server of the resource?